ush/python/setup.cfg

```diff
@@ -0,0 +1,62 @@
+[metadata]
+name = pygw
+version = 0.0.1
+description = Global applications specific workflow related tools
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = "NOAA/NCEP/EMC"
```

Comment:
don't forget the National Weather Service!

Reply:
done.